gears/src/application/handlers.rs

```diff
@@ -141,7 +158,7 @@ pub trait AuxHandler {
 }
 
 // TODO: we're assuming here that the app has an auth module which handles this query
-fn get_account_latest(address: AccAddress, node: &str) -> anyhow::Result<QueryAccountResponse> {
+pub fn get_account_latest(address: AccAddress, node: &str) -> anyhow::Result<QueryAccountResponse> {
```

Comment:
Does this need to be public?

Reply:
I checked the code, I used it at least twice. 

Here I restored the original version. I'll use the alternative implementation in `ultradao`.